inception-active-learning/src/main/java/de/tudarmstadt/ukp/inception/active/learning/sidebar/ActiveLearningSidebar.java

```diff
@@ -529,7 +530,7 @@ private void jumpAndHighlightFromLearningHistory(AjaxRequestTarget aTarget,
     {
         actionShowSelectedDocument(aTarget, record.getSourceDocument(),
             record.getOffsetCharacterBegin());
-        JCas aJcas = this.getJCasProvider().get();
+        jCas = this.getJCasProvider().get();
```

Comment:
The CAS cannot be a field in a Wicket component - same reason -> it is not serializable. It must be obtained in the methods where it is needed.